install.sh

```diff
@@ -257,6 +257,8 @@ set_synchrony_url() {
     SYNCHRONY_FULL_URL=$(terraform output | sed "s/ //g" | grep "synchrony_url=" | sed -nE 's/^.*"(.*)".*$/\1/p')
     helm upgrade confluence atlassian-data-center/confluence -n atlassian --reuse-values --set synchrony.ingressUrl="${SYNCHRONY_FULL_URL}" > /dev/null
     log "Synchrony URL is set to '${SYNCHRONY_FULL_URL}'."
+    echo
+    helm list -n atlassian
```

Comment:
This will happen only if confluence is installed and doesn't have a domain. I think we can print the helm releases every time. I think it deserves its own function at the end of execution.

Reply:
Right, I moved this after if block.